platform/broadcom/sonic-platform-modules-accton/as7726-32x/sonic_platform/fan.py

```diff
@@ -15,5 +18,25 @@ def __init__(self, tray_idx, fan_idx=0, pddf_data=None, pddf_plugin_data=None, i
         PddfFan.__init__(self, tray_idx, fan_idx, pddf_data, pddf_plugin_data, is_psu_fan, psu_index)
 
     # Provide the functions/variables below for which implementation is to be overwritten
-    # Since AS4630 psu_fan airflow direction cant be read from sysfs, it is fixed as 'F2B' or 'intake'
+    
+    def get_position_in_parent(self):
+        """
+        Retrieves 1-based relative physical position in parent device.
+        If the agent cannot determine the parent-relative position
+        for some reason, or if the associated value of
+        entPhysicalContainedIn is'0', then the value '-1' is returned
+        Returns:
+            integer: The 1-based relative physical position in parent device
+            or -1 if cannot determine the position
+        """
+        return (self.fan_index) \
+            if not self.is_psu_fan else (self.fans_psu_index+1)
```

Comment:
(self.fans_psu_index+1)
This is wrong. 
fans_psu_index = PSU index (1-based) of the PSU-fan.
so,  
For PSU1 it will calculate to 2 and for PSU2 it will be 3.